app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -68,6 +70,7 @@
     private static final String KEY_SEND_INFO_LEVEL = "key_send_info_level";
     private static final String KEY_SEND_SCREENSHOT_DIR = "key_send_screenshot_dir";
 
+    private static final String PREF_USER_CORRECTIONS = "user_corrections";
```

Comment:
Based on docs in https://developer.android.com/training/basics/data-storage/shared-preferences.html#GetSharedPreferences, we should use here `com.kamron.pogoiv.USER_CORRECTIONS` as name, IIUC to avoid conflicts. Can you fix that?

Otherwise the code LGTM! 👍